vllm/engine/arg_utils.py

```diff
@@ -504,12 +504,14 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
                   'Enabling this will use the fully sharded layers. '
                   'At high sequence length, max rank or '
                   'tensor parallel size, this is likely faster.'))
+        # yapf: disable
         parser.add_argument(
             "--device",
             type=str,
             default=EngineArgs.device,
-            choices=["auto", "cuda", "neuron", "cpu", "tpu", "xpu"],
+            choices=["auto", "cuda", "neuron", "cpu", "openvino", "tpu", "xpu"],
             help='Device type for vLLM execution.')
+        # yapf: enable
```

Comment:
Why is yapf disabled here? Did you have issues with `./format.sh`?

Reply:
I have not used it 😄 

With such script formatting is really simple, thank you for pointing out.